presto-oracle/src/test/java/io/prestosql/plugin/oracle/TestOraclePoolIntegrationSmokeTest.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.oracle;
+
+import io.prestosql.testing.QueryRunner;
+import io.prestosql.tpch.TpchTable;
+
+public class TestOraclePoolIntegrationSmokeTest
+        extends TestOracleIntegrationSmokeTest
```

Comment:
Please extract abstract `BaseOracleIntegrationSmokeTest` with all the test changes needed for Oracle, but with `createQueryRunner` not implemented.
And then derive `TestOraclePoolIntegrationSmokeTest` and `TestOracleIntegrationSmokeTest` from it.
With that we do not extend non-abstract class. And we can get rid of `createOracleQueryRunner`.

Reply:
@losipiuk I left the createOracleQueryRunner, so initialization of server and table definition could be on BaseTest as this are equal to both implementations